src/openms/source/APPLICATIONS/TOPPBase.cpp

```diff
@@ -1596,21 +1596,10 @@ namespace OpenMS
       writeLog_("Exit code: " + String(qp.exitCode()));
       if (any_failure)
       {
-        qp.close();
         return EXTERNAL_PROGRAM_ERROR;
       }
     }
 
-    if (debug_level_ >= 10)
```

Comment:
would it make sense to put it into the any failure block?

Reply:
it was already .. see some lines above.